src/static/styles/App.scss

```diff
@@ -1,624 +1,764 @@
+    box-shadow: 0rem 0.4475rem 0.85rem 0rem rgba(0, 0, 0, 0.2);
 }
 
 .no-link-style {
-	color: inherit;
-	text-decoration: none;
+    color: inherit;
+    text-decoration: none;
 }
 
 .no-link-style:hover {
-	color: inherit;
-	text-decoration: none;
+    color: inherit;
+    text-decoration: none;
 }
 
 .pie-chart-dimensions {
-	height: 40vh;
-	width: 40vh;
-	min-width: 12.5rem;
-	min-height: 12.5rem;
+    height: 40vh;
+    width: 40vh;
+    min-width: 12.5rem;
+    min-height: 12.5rem;
 }
 
 .icon-svg {
-	height: 1.5rem;
-	width: 1.5rem;
+    height: 1.5rem;
+    width: 1.5rem;
 }
 
 .primary-color-border {
-	border-color: $primary-theme;
-	border-radius: 0.25rem;
-	border-width: 0.0625rem;
-}
-
-@mixin scrollbars($size, $front-color, $back-color) {
-	// For Google Chrome
-	&::-webkit-scrollbar {
-		width: $size;
-		height: $size;
-	}
-
-	&::-webkit-scrollbar-thumb {
-		background: $front-color;
-	}
-
-	&::-webkit-scrollbar-track {
-		background: $back-color;
-	}
-
-	// For Internet Explorer
-	& {
-		scrollbar-face-color: $front-color;
-		scrollbar-track-color: $back-color;
-	}
+    border-color: $primary-theme;
+    border-radius: 0.25rem;
+    border-width: 0.0625rem;
+}
+
+@mixin scrollbars($size,
+$front-color,
+$back-color) {
+    // For Google Chrome
+    &::-webkit-scrollbar {
+        width: $size;
+        height: $size;
+    }
+    &::-webkit-scrollbar-thumb {
+        background: $front-color;
+    }
+    &::-webkit-scrollbar-track {
+        background: $back-color;
+    }
+    // For Internet Explorer
+    & {
+        scrollbar-face-color: $front-color;
+        scrollbar-track-color: $back-color;
+    }
 }
 
 div.scrollbar {
-	@include scrollbars(0.75rem, $primary-theme, #ffffff);
+    @include scrollbars(0.75rem,
+    $primary-theme,
+    #ffffff);
 }
 
 .lead-medium {
-	font-size: 1rem;
+    font-size: 1rem;
 }
 
 .profile-picture {
-	width: 10vw;
-	min-width: 10.5rem;
-	border-radius: 0.25rem;
+    width: 10vw;
+    min-width: 10.5rem;
+    border-radius: 0.25rem;
 }
 
 .modal-backdrop.fade {
-	opacity: .25;
-	filter: alpha(opacity=25);
+    opacity: .25;
+    filter: alpha(opacity=25);
 }
 
 .modal-backdrop.fade.in {
-	opacity: 0.5;
-	filter: alpha(opacity=50);
+    opacity: 0.5;
+    filter: alpha(opacity=50);
 }
 
+// Table styling
+// class for rows being edted
+.table-edit-row {
+    background-color: $primary-theme-light;
+    font-family: 'Inter', sans-serif;
+    font-weight: 400;
+    //color: $secondary-dark-theme;
+}
 
+.table-zebra {
+    background: #F4F4F4;
+    //box-shadow: inset 0px -1px 0px #CECECE;
+}
 
-// Table styling
+.table-zebra:hover,
+.table-row:hover {
+    background: $primary-theme-light !important;
+}
 
+.editor-input {
+    font-family: 'Inter', sans-serif;
+    font-size: 0.9rem;
+    text-align: left;
+    font-weight: 400 !important;
+    color: $secondary-dark-theme !important;
+    background-color: white !important;
+    margin: auto;
+}
 
-// class for rows being edted
```

Comment:
can you move all table scss into a different scss file (and then import it into app) before things get too confusing?

Reply:
Yes! Based on [this](https://stackoverflow.com/questions/17598996/sass-use-variables-across-multiple-files) and [this](https://sass-lang.com/documentation/at-rules/use) and [this](https://github.com/material-components/material-components-web/issues/6431), @import will eventually be deprecated -> should use @use but we don't have Dart Sass so that doesn't work yet.

ideally we’d use @use ‘App’ and access variables like App.$color-name; otherwise everything is global / a bit confusing